.devcontainer/Dockerfile

```
@@ -0,0 +1,16 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# To fully customize the contents of this image, use the following Dockerfile instead:
+# https://github.com/microsoft/vscode-dev-containers/tree/v0.106.0/containers/typescript-node-12/.devcontainer/Dockerfile
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-12
```

```suggestion
FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:12
```

At least according to https://hub.docker.com/_/microsoft-vscode-devcontainers

---

This file was automatically created by VS Code. The leading zero seems to indicate the major version of the image (see https://github.com/microsoft/vscode-dev-containers/blob/master/containers/typescript-node-12/definition-build.json#L7)